packages/SketchMorph2-Tests.package/M2DrawingComponentTests.class/instance/drawDiagonalCirclesFrom.to..st

```diff
@@ -6,4 +6,4 @@ drawDiagonalCirclesFrom: aStartNumber to: anEndNumber
 			userActionMoveDrawing: ((4 * x) + 2 ) asPoint;
 			userActionStopDrawing.
 	]
```

Comment:
I'm not sure, but I don't think that is the squeak-like way of closing blocks.

Reply:
As above.